health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/pages/employee/PopInbox.js

```
@@ -560,60 +559,59 @@ const PopInbox = () => {
-
-                          const isPrimary = getButtonState(action.action);
-
-                          return (
+        <div className="pop-inbox-wrapper-filter-table-wrapper" style={{ marginBottom: (isRootApprover && isStatusConditionMet(totalStatusCount) && planObject?.status === "CENSUS_DATA_APPROVAL_IN_PROGRESS") || (!isRootApprover && isStatusConditionMet(totalStatusCount)) || disabledAction ? "2.5rem" : "0rem" }}>
+          <InboxFilterWrapper
+            options={activeFilter}
+            onApplyFilters={onFilter}
+            clearFilters={clearFilters}
+            defaultValue={ { [selectedFilter]: activeFilter[selectedFilter]} 
+      }
+          ></InboxFilterWrapper>
```

Biome

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)